architecture/mce-overview-ocp.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+[id="mce-overview-ocp"]
+= About multicluster engine for Kubernetes operator
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: mce-overview-ocp
+
+toc::[]
+
+One of the challenges of scaling Kubernetes environments is managing the lifecycle of a growing fleet. To meet that challenge, you can use multicluster engine for Kubernetes operator (MCE). The operator delivers full lifecycle capabilities for managed OpenShift clusters and partial lifecycle management for other Kubernetes distributions. It is available in two ways:
+
+* As a standalone operator that you install as part of your OpenShift or Red Hat OpenShift Kubernetes Engine subscription
+* As part of link:https://access.redhat.com/products/red-hat-advanced-cluster-management-for-kubernetes[Red Hat Advanced Cluster Management for Kubernetes]
+
+[id="mce-on-ocp"]
+== Cluster management with multicluster engine on {product-title}
+
+When you enable multicluster engine on {product-title}, you gain the following capabilities:
+
+* xref:control-plane.adoc#hosted-control-planes-overview_control-plane[Hosted control planes], which is based on the HyperShift project. With hosted control planes, you have a centralized control plane for OpenShift clusters so that you can operate clusters in a hyperscale manner.
```

Comment:
I'd recommend either "Hosted control planes, which are based on..." or "Hosted control planes, which is a feature based on..." 

Also, is "hyperscale" a well known technical term? It's new to me and I'm wondering if you should either use a more generic word or define "hyperscale".  Just a couple of wording suggestions and replacing "openshift" with the variable.

```suggestion
* xref:control-plane.adoc#hosted-control-planes-overview_control-plane[Hosted control planes], which are based on the HyperShift project. Centralized hosted control planes let you operate {product-title} clusters in a hyperscale manner.
```

Reply:
Good feedback here! 

I think I'll use "Hosted control planes, which is a feature that is based on the HyperShift project." This is one of those tricky feature names  because technically it's singular, but it sounds like we're talking about multiple things. 

I had the same thought about "hyperscale" being not well known enough, but I did some searching, and it looks like it's a term that's being adopted in the industry. I don't find it mentioned in the ISG, SSG, or the OpenShift glossary, but then again, those sources don't define every technical term. I'm tempted to leave it for now unless we get more feedback that it's unclear. 

I like your suggested rewrite, but I might tweak it a bit to avoid using the phrase "hosted control planes let you...". The ISG warns against phrases like "lets you..." or "allows you..." because by using those phrases, we're assigning human characteristics to inanimate objects (see [Anthropomorphism](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism)). 

I'm thinking of changing it to something like, "Hosted control planes, which is a feature that is based on the HyperShift project. With a centralized hosted control plane, you can operate {product-title} clusters in a hyperscale manner."

Reply:
This sounds great to me!